src/App.vue

```diff
@@ -212,24 +206,24 @@ export default class App extends Vue {
   @Action(useStore) setFilingId!: ActionBindingIF
 
   // Local properties
-  protected accountAuthorizationDialog = false
-  protected confirmDeleteAllDialog = false
-  protected fetchErrorDialog = false
-  protected fileAndPayInvalidNameRequestDialog = false
-  protected nameRequestErrorDialog = false
-  protected nameRequestErrorType = ''
-  protected paymentErrorDialog = false
-  protected saveErrorDialog = false
-  protected saveErrors: Array<object> = []
-  protected saveWarnings: Array<object> = []
-  protected staffPaymentErrorDialog = false
+  accountAuthorizationDialog = false
+  confirmDeleteAllDialog = false
+  fetchErrorDialog = false
+  fileAndPayInvalidNameRequestDialog = false
+  nameRequestErrorDialog = false
+  nameRequestErrorType = ''
+  paymentErrorDialog = false
+  saveErrorDialog = false
+  saveErrors: Array<object> = []
+  saveWarnings: Array<object> = []
+  staffPaymentErrorDialog = false
```

Comment:
I made these (and methods below) public so the template can see them (including types).